src/Sulu/Component/PHPCR/NodeTypes/Content/ContentNodeType.php

```diff
@@ -49,6 +49,7 @@ public function getDeclaredSupertypeNames()
     {
         return [
             'sulu:base',
+            'mix:versionable',
```

Comment:
Don't include mixin directly in NodeType, otherwise it won't be usable in PHPCR implementations without versioning.